html/changelogs/Kurfurst.yml

```diff
@@ -0,0 +1,14 @@
+author: Kurfurst
+delete-after: True
+changes:
+ - tweak: You can now repair the radio transmitter with a solder
+ - tweak: Radio transmitter is now dense (you can push it)
+ - tweak: Fixed a bug in which you couldn't examine someone's ear slot

```

Comment:
If you actually are going to consider these bugs use the -bugfix: moniker instead of tweak